backend/src/main/java/com/htecgroup/SkyNest/model/response/ErrorMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.htecgroup.SkyNest.model.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
do you need `@Setter ` ?

Reply:
I think not, because this class will only be used at the ControllerAdvice level. Will remove it.